### ISSUE TITLE
Bikeshed update does not work anymore?

### ISSUE THREAD

**Issue by FremyCompany** [tabatkins/bikeshed#909]
I had to reinstall my windows from scratch, but it seems that ```bikeshed update``` doesn't work anymore.

Here is the error message I get:
```
Either this version of the anchor-data API is no longer supported, or (more likely) there was a transient network error. Try again in a little while, and/or update Bikeshed. If the error persists, please report it on GitHub.
```

Here is the the only thing bikeshed requests on the network:
``` 
GET https://api.csswg.org/shepherd/ HTTP/1.1
Accept-Encoding: identity
Host: api.csswg.org
Connection: close
Accept: application/home+json, application/json-home, application/json
User-Agent: Python-urllib/2.7
```
```
{
    "resources": {
        "specifications": {
            "href-template": "https://test.csswg.org/shepherd/api/spec/{?spec,sections,anchors,draft,insync}",
            "href-vars": {
                "spec": "param/spec",
                "sections": "param/spec/sections",
                "anchors": "param/spec/anchors",
                "draft": "param/spec/draft",
                "insync": "param/spec/insync"
            },
            "hints": {
                "allow": [
                    "GET"
                ],
                "formats": {
                    "text/vnd.csswg.shepherd.v1+javascript": {},
                    "application/vnd.csswg.shepherd.v1+json": {},
                    "text/javascript": {},
                    "application/json": {}
                },
                "docs": "https://test.csswg.org/shepherd/api/"
            }
        },
        "technical_report": {
            "href-template": "https://test.csswg.org/shepherd/api/tr/{?spec, group}",
            "href-vars": {
                "spec": "param/spec",
                "group": "param/group"
            },
            "hints": {
                "allow": [
                    "GET"
                ],
                "formats": {
                    "text/vnd.csswg.shepherd.v1+javascript": {},
                    "application/vnd.csswg.shepherd.v1+json": {},
                    "text/javascript": {},
                    "application/json": {}
                },
                "docs": "https://test.csswg.org/shepherd/api/"
            },
            "functions": {
                "getSpec": {
                    "arguments": [
                        "spec"
                    ],
                    "method": "GET",
                    "format": "application/json"
                },
                "getSpecsForGroup": {
                    "arguments": [
                        "group"
                    ],
                    "method": "GET",
                    "format": "application/json"
                }
            }
        },
        "test_flags": {
            "href-template": "https://test.csswg.org/shepherd/api/flag/{?flag}",
            "href-vars": {
                "flag": "param/flag"
            },
            "hints": {
                "allow": [
                    "GET"
                ],
                "formats": {
                    "text/vnd.csswg.shepherd.v1+javascript": {},
                    "application/vnd.csswg.shepherd.v1+json": {},
                    "text/javascript": {},
                    "application/json": {}
                },
                "docs": "https://test.csswg.org/shepherd/api/"
            }
        },
        "test_formats": {
            "href-template": "https://test.csswg.org/shepherd/api/format/{?format}",
            "href-vars": {
                "format": "param/format"
            },
            "hints": {
                "allow": [
                    "GET"
                ],
                "formats": {
                    "text/vnd.csswg.shepherd.v1+javascript": {},
                    "application/vnd.csswg.shepherd.v1+json": {},
                    "text/javascript": {},
                    "application/json": {}
                },
                "docs": "https://test.csswg.org/shepherd/api/"
            }
        },
        "test_suites": {
            "href-template": "https://test.csswg.org/shepherd/api/suite/{?suite,spec}",
            "href-vars": {
                "repo": "param/repo",
                "suite": "param/suite",
                "spec": "param/spec"
            },
            "hints": {
                "allow": [
                    "GET"
                ],
                "formats": {
                    "text/vnd.csswg.shepherd.v1+javascript": {},
                    "application/vnd.csswg.shepherd.v1+json": {},
                    "text/javascript": {},
                    "application/json": {}
                },
                "docs": "https://test.csswg.org/shepherd/api/"
            }
        }
    }
}
```

Anything I can do to make this work again? I tried using an older version but that doesn't seem to help at least not the few ones I tried.

**Comment by tabatkins** [comment]
I literally just used it, so it's nothing wrong in general. ^_^

That message is gated by `if ((not res) or (406 == res.status)):`, so one of those two happened.

**Comment by FremyCompany** [comment]
Weird, I definitely do see the request succeeding through Fiddler. 

Thanks for quick reply, I will investigate further...

**Comment by FremyCompany** [comment]
Dug this deeper, and actually the issue seemed to be I didnt follow your instructions properly, and had installed the 64bit version of LXML (which is of course not compatible with the 32bit version of python that instructions have us install). It is working fine now.

**Comment by tabatkins** [comment]
...I'm confused as to how *that* caused this error, but ok, glad you got yourself fixed.

**Comment by herrjemand** [comment]
@tabatkins I think anchor-data api is down https://api.csswg.org/

**Comment by tabatkins** [comment]
It appears to be working for me.  What error are you getting?  (And can you report it in a new issue?)
